modules/cnf-scheduling-numa-aware-workloads.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-topology-aware-scheduling.adoc
+
+[id="cnf-scheduling-numa-aware-workloads_{context}"]
+= Scheduling topology-aware workloads
+
+You can schedule topology-aware workload deployments using `Deployment` CRs that specify the minimum required resources to process the workload.
+
+The following example deployment uses topology-aware scheduling for a sample workload.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Get the name of the topology-aware scheduler that is deployed in the cluster:
++
+[source,terminal]
+----
+$ oc get numaresourcesschedulers.nodetopology.openshift.io numaresourcesscheduler -o json | jq '.status.schedulerName'
+----
++
+.Example output
+[source,terminal]
+----
+topo-aware-scheduler
+----
+
+. Create a `Deployment` CR that uses the `secondary-scheduler`, for example:
+
+.. Save the following YAML in the `nro-deployment.yaml` file:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: numa-deployment-1
+  namespace: openshift-numaresources-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: test
+  template:
+    metadata:
+      labels:
+        app: test
+    spec:
+      schedulerName: topo-aware-scheduler
+      containers:
+      - name: ctnr
+        image: quay.io/openshifttest/hello-openshift:openshift
+        imagePullPolicy: IfNotPresent
+        resources:
+          limits:
+            memory: "100Mi"
+            cpu: "10"
+          requests:
+            memory: "100Mi"
+            cpu: "10"
+      - name: ctnr2
+        image: quay.io/fromani/numalign
```

Comment:
let's use a different example in the final example :)

Reply:
Yes indeed :)

Reply:
Can you suggest an example image. Is there something added to cnf-tests maybe?

Reply:
a safe fallback is the `pause` image: gcr.io/google_containers/pause-amd64:3.0